tests/layer_tests/common/tf_layer_test_class.py

```diff
@@ -61,18 +89,48 @@ def get_framework_results(self, inputs_dict, model_path):
                 sess.graph.as_default()
                 tf.compat.v1.import_graph_def(graph_def, name='')
 
-                input = dict()
-                for key in inputs_dict.keys():
-                    data = inputs_dict.get(key)
-                    if self.use_old_api or self.use_new_frontend:
-                        key += ':0'
-                    input[key] = transpose_nchw_to_nhwc(data, self.use_new_frontend, self.use_old_api)
-
-                tf_res = sess.run([out + ":0" for out in outputs_list], input)
```

Comment:
why did you remove it? Will it affect TF layer tests?

Reply:
I didn't remove it, please check https://github.com/openvinotoolkit/openvino/blob/8e0e6983124a0238418dda6cd430623132ce5222/tests/layer_tests/common/tf_layer_test_class.py#L92